integration.md

```diff
@@ -218,5 +218,5 @@ Attribute | Description
 data-api-key | This is where you must supply your API key for the form to function properly.
 data-flow | The type of on-demand form to display. The supported values are `purchase` and `recurring-purchase`
 data-locale | The locale in which the form should be presented.
-data-amount | This attribute is valid only for purchase forms and denotes the purchase's total price.
+data-amount | This attribute is valid only for purchase forms and denotes the purchase's total price. The value should be multiplied by 100, so if your price is "9,90" it should be sent as "990".
```

Comment:
You can say that it should be in the ISO 4217 (https://en.wikipedia.org/wiki/ISO_4217) format and in this example you should multiply by 100